### PR TITLE
store-against-0.3.49

### PR DESCRIPTION
**Pull request by yeelali14**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=DnAsJTG)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,4 +74,7 @@ And in the free fall I will realize
 I'm better off when I hit the bottom
 
 Lady gaga the queen
+Lady gaga the queen
+Lady gaga the queen
+Lady gaga the queen
```
Comment (ShakedZrihen) [inline README.md:79]: always good to add lady gaga

**Review by ShakedZrihen — APPROVED** [review]
LG to me

File: README.md [hunk 2]
```diff
@@ -74,4 +74,7 @@ And in the free fall I will realize
 I'm better off when I hit the bottom
 
 Lady gaga the queen
+Lady gaga the queen
+Lady gaga the queen
```
Comment (EladKohavi) [inline README.md:78]: numnum

File: README.md [hunk 3]
```diff
@@ -74,4 +74,7 @@ And in the free fall I will realize
 I'm better off when I hit the bottom
```
Comment (EladKohavi) [inline README.md:74]: nice

**Review by EladKohavi — APPROVED** [review]

**Comment by EladKohavi** [comment]
ok

**Pull request closed without merging**